### PR TITLE
Fixes #8804: Give more precise requirements for JVM to manage AIX sha256/512 passwords

### PR DESCRIPTION
**Pull request by fanf**
https://www.rudder-project.org/redmine/issues/8804

### CODE REVIEW

File: 4_advanced_usage/50_script_in_directives.txt [hunk 1]
```diff
@@ -116,10 +116,12 @@ at each policy generation.
 
 ====
 
-AIX password generation depends on the availability of *PBKDF2WithHmac256* and
-*PBKDF2WithHmac512* in the JVM. Some JVMs don't include these libraries. In
-this case, the hashing algorithm falls back to *SHA1*, and an error message
-will be logged.
+AIX password generation depends on the availability of *PBKDF2WithHmacSHA256* and
+*PBKDF2WithHmacSHA512* in the JVM. These algorithms are included by default on 
+HotSpot 1.8 and OpenJDK 1.8 and upward. In the case where your JVM does not support theses
```
Comment (jooooooon) [inline 4_advanced_usage/50_script_in_directives.txt]: s/theses/these/

File: 4_advanced_usage/50_script_in_directives.txt [hunk 2]
```diff
@@ -116,10 +116,12 @@ at each policy generation.
...
+algorithms, typically on a JVM JDK or JVM 1.7 version, the hashing algorithm falls back to
```
Comment (jooooooon) [inline 4_advanced_usage/50_script_in_directives.txt]: I don't understand the bit "JVM JDK or JVM 1.7". In my understanding, "JVM *" is a subset of "JVM", so why repeat this?

Comment (fanf) [inline 4_advanced_usage/50_script_in_directives.txt]: Ah, should have been "on an IBM JDK or JVM 1.7"

**Comment by fanf** [comment]
PR rebased

File: 4_advanced_usage/50_script_in_directives.txt [hunk 3]
```diff
@@ -116,10 +116,12 @@ at each policy generation.
...
 ====
 
-AIX password generation depends on the availability of *PBKDF2WithHmac256* and
-*PBKDF2WithHmac512* in the JVM. Some JVMs don't include these libraries. In
-this case, the hashing algorithm falls back to *SHA1*, and an error message
-will be logged.
+AIX password generation depends on the availability of *PBKDF2WithHmacSHA256* and
+*PBKDF2WithHmacSHA512* in the JVM. These algorithms are included by default on 
+HotSpot 1.8 and OpenJDK 1.8 and upward. In the case where your JVM does not support these
+algorithms, typically on an IBM JDK or a JVM 1.7 version of HotStop and OpenJDK, the hashing
```
Comment (jooooooon) [inline 4_advanced_usage/50_script_in_directives.txt]: s/HotSpot/HotSpot/ (or the opposite on the previous line)

**Comment by fanf** [comment]
PR rebased

**Pull request merged**